shiftmailer/excelexport.py

```diff
@@ -1,61 +1,148 @@
 # coding: utf-8
-
-import time
+import logging
+import os
+import tempfile
 
 from django.core.mail.message import EmailMessage
 import xlwt
-from django.db.models import Count
+
+log = logging.getLogger(__name__)
```

Comment:
This is a nitpick, but `logger = logging.getLogger(__name__)` is, if I'm not mistaken, the much more common pattern.